Makefile

```diff
@@ -123,15 +114,6 @@ variables:
 	@echo "DESTDIR: $(DESTDIR)"
 	@echo "AVOCADO_DIRNAME: $(AVOCADO_DIRNAME)"
 	@echo "AVOCADO_OPTIONAL_PLUGINS: $(AVOCADO_OPTIONAL_PLUGINS)"
-	@echo "RELEASE_COMMIT: $(RELEASE_COMMIT)"
-	@echo "RELEASE_SHORT_COMMIT: $(RELEASE_SHORT_COMMIT)"
-	@echo "COMMIT: $(COMMIT)"
-	@echo "COMMIT_DATE: $(COMMIT_DATE)"
-	@echo "SHORT_COMMIT: $(SHORT_COMMIT)"
-	@echo "MOCK_CONFIG: $(MOCK_CONFIG)"
-	@echo "ARCHIVE_BASE_NAME: $(ARCHIVE_BASE_NAME)"
-	@echo "PYTHON_MODULE_NAME: $(PYTHON_MODULE_NAME)"
-	@echo "RPM_BASE_NAME: $(RPM_BASE_NAME)"
 
 propagate-version:
 	for DIR in $(AVOCADO_OPTIONAL_PLUGINS); do\
```

Comment:
If after discussion about my previous point, we decide to go this way, I won't mind if this gets squashed with the previous commit.